qa/tasks/cephfs/test_admin.py

```diff
@@ -2211,6 +2211,10 @@ def test_with_health_warn_oversize_cache(self):
                               retval=1, errmsgs=errmsg)
         self.run_ceph_cmd(f'fs fail {self.fs.name} --yes-i-really-mean-it')
 
+        # so that unmounting can happen successfully
+        self.fs.set_joinable()
+        self.fs.wait_for_daemons()
```

Comment:
Do we still need force/lazy umount after we waiting for the daemons?

Reply:
Nope, we don't.